src/components/order-service/order-service.controller.ts

```diff
@@ -183,4 +190,14 @@ export class OrderServiceController implements angular.IController {
   private isNewProject(): boolean {
     return this.ctrl.selectedProject && !_.has(this.ctrl.selectedProject, 'metadata.uid');
   }
+  private watchResults = (resource: any, data: any, context: any) => {
+    this.watches.push(this.DataService.watchObject(resource, data.metadata.name, context, (instanceData: any, action: any) => {
+      var conditions: any = _.get(instanceData, 'status.conditions');
+
+      this.ctrl.orderComplete = _.find(conditions, {type: "Ready"});
+      var errorCondition: any = _.find(conditions, {status: "False"});
```

Comment:
the existence of the Ready condition doesn't mean the order is complete.  There will be a point in time where Ready will have status False, not because it has failed, but because it is still provisioning.  I would definitely not assume the existence of a False condition means that there is an error.